vortexasdk/endpoints/cargo_movements.py

```diff
@@ -78,6 +79,10 @@ def search(
 
             filter_waypoints: A geography ID, or list of geography IDs to filter on.
 
+            exclude: A dictionary where keys are strings that represent the parameter to implement exclusions on and
+                values are IDs or list of IDs that correspond to the entities to be exluded. Dictionary keys must be one of
+                ['filter_origins', 'filter_destinations', 'filter_products', 'filter_vessels', 'filter_charterers', 'filter_owners']
+
```

Comment:
Is that clear enough? Should we maybe add or modify any current example to illustrate the exact use of this filter?

Reply:
Similarly we can have:
```
CargoMovements.search(filter_origins=us, exclude_origins=padd1])
```

this keeps it simple and explicit for the user, avoiding nesting.